<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeSelectWidget</class>
 <widget class="QWidget" name="DateTimeSelectWidget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择时间</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="wid_time_select" native="true">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
   <widget class="QSingleSelectTimeWidget" name="wid_hour_select" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QSingleSelectTimeWidget" name="wid_minute_select" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>41</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QSingleSelectTimeWidget" name="wid_second_select" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>41</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btn_ok">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_curDate">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2025-03-04 20:01:03</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSingleSelectTimeWidget</class>
   <extends>QWidget</extends>
   <header>qsingleselecttimewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
